hummingbot/strategy/hanging_orders_tracker.py

```diff
@@ -98,31 +225,25 @@ def remove_all_sells(self):
         for order in to_be_removed:
             self.original_orders.remove(order)
 
+    def hanging_order_age(self, hanging_order: HangingOrder) -> float:
+        """
+        Returns the number of seconds between the current time (taken from the strategy) and the order creation time
+        """
+        return (self.strategy.current_timestamp - hanging_order.creation_timestamp
+                if hanging_order.creation_timestamp
+                else -1)
+
     def renew_hanging_orders_past_max_order_age(self):
-        max_age = getattr(self.strategy, "max_order_age", None)
         to_be_cancelled: Set[HangingOrder] = set()
-        to_be_created: Set[HangingOrder] = set()
-        if max_age:
+        max_order_age = getattr(self.strategy, "max_order_age", lambda: None)
+        if max_order_age:
```

Comment:
In case max_order_age is lambda: None this condition will also be True

Reply:
Good catch Nico, thank you for that.